packages/react-experiments/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.test.tsx

```diff
@@ -4,20 +4,16 @@ import { create } from 'react-test-renderer';
 import { people } from '@fluentui/example-data';
 import { mount } from 'enzyme';
 import { SelectedPeopleList, SelectedPersona, ItemWithContextMenu, TriggerOnContextMenu } from '../index';
-import type { ISelectedPeopleList, ItemCanDispatchTrigger } from '../index';
+import type { ItemCanDispatchTrigger } from '../index';
 
 describe('SelectedPeopleList', () => {
   it('renders nothing if nothing is provided', () => {
-    const pickerRef: React.RefObject<ISelectedPeopleList> = React.createRef();
```

Comment:
This ref was causing a build error due to a slight typing mismatch. Since it wasn't used at all in the test, I removed it.